github/pulls.go

```diff
@@ -62,6 +62,8 @@ type PullRequest struct {
 
 	Head *PullRequestBranch `json:"head,omitempty"`
 	Base *PullRequestBranch `json:"base,omitempty"`
+
+	ActiveLockReason *string `json:"active_lock_reason,omitempty"`
```

Comment:
Could you please add a descriptive comment here as well with the possible values?